modules/docs/arrow-docs/docs/docs/typeclasses/intro/README.md

```diff
@@ -86,9 +86,13 @@ We will list them by their hierarchy.
 
 - [`MonoidK`]({{ '/docs/typeclasses/monoidk/' | relative_url }}) - combinable datatypes have an empty value
 
-##### Functor
+##### Invariant, Contravariant, Functor
 
-- [`Functor`]({{ '/docs/typeclasses/functor/' | relative_url }}) - its contents can be mapped
+- [`Invariant`]({{ '/docs/typeclasses/Invariant/' | relative_url }}) - its contents can be bidirectionally mapped
+
+- [`Contravariant`]({{ '/docs/typeclasses/contravariant/' | relative_url }}) - its contents can be mapped reverse sequentially
```

Comment:
To be honest I'd move these below `Bimonad`, as they're not as common as the others. Same in the menu.

Reply:
The intro stated that "We will list them by their hierarchy" so I used that as the guide, however if commonality is more important, then I can switch them around, remove the quote, and change the section title back to Functor.

Reply:
Fair, up to you then :D